packages/shared/src/lib/core/i18n/utils/getDecimalSeparator.ts

```diff
@@ -12,7 +12,7 @@ export function getDecimalSeparator(currency: string | undefined = undefined): s
     return (
         Intl.NumberFormat(appLanguage, {
             style: 'currency',
-            currency: currency ?? 'USD',
+            currency: currency.length > 3 ? 'USD' : currency,
```

Comment:
what exactly does `currency.length > 3` mean? could you extract this into a variable to explain this expression?

Reply:
+ tests are failing on a `length of undefined`